lib/foreman_remote_execution/engine.rb

```diff
@@ -45,6 +45,8 @@ class Engine < ::Rails::Engine
                                             :'api/v2/job_templates' => [:update],
                                             :'api/v2/template_inputs' => [:create, :update, :destroy],
                                             :'api/v2/foreign_input_sets' => [:create, :update, :destroy]}, :resource_type => 'JobTemplate'
+          permission :edit_remote_execution_features, { :remote_execution_features => [:index, :show, :update],
```

Comment:
not needed now but would it make sense to split into two permissions, one for viewing, second for editing (the form would either display select box or just text information)? if you agree, creating a tracking issue would be enough